libs/wallet/src/connect-dialog/connect-dialog.tsx

```diff
@@ -306,77 +300,162 @@ const SelectedForm = ({
     );
   }
 
-  if (connector instanceof ViewConnector) {
-    return (
-      <ViewConnectorForm
-        connector={connector}
-        onConnect={onConnect}
-        reset={reset}
-      />
-    );
-  }
-
   throw new Error('No connector selected');
 };
 
+const GetWallet = () => {
+  const isItChrome = window.navigator.userAgent.includes('Chrome');
+  const isItMozilla =
+    window.navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
+  const onClick = () => {
+    if (isItMozilla) {
+      window.open(MOZILLA_EXTENSION_URL, '_blank');
+      return;
+    }
+    if (isItChrome) {
+      window.open(CHROME_EXTENSION_URL, '_blank');
+    }
+  };
+
+  const buttonContent = (
+    <>
+      <div className="flex items-center justify-center gap-1 text-base">
+        {t('Get the Vega Wallet')}
+        <Lozenge className="text-[10px] !font-alpha bg-vega-blue-500 dark:bg-vega-blue-500 py-0 px-1">
+          ALPHA
+        </Lozenge>
+      </div>
+      <BrowserIcon />
+    </>
+  );
+
+  return !isItChrome && !isItMozilla ? (
+    <div
+      className={classNames([
+        'bg-vega-blue-350 hover:bg-vega-blue-400 dark:bg-vega-blue-650 dark:hover:bg-vega-blue-600',
+        'flex gap-2 items-center justify-center rounded h-8 px-3 relative',
+      ])}
+      data-testid="get-wallet-button"
+    >
+      {buttonContent}
+    </div>
+  ) : (
+    <TradingButton
+      onClick={onClick}
+      intent={Intent.Info}
+      data-testid="get-wallet-button"
+      className="!block w-full relative"
```

Comment:
We shouldn't need to override the styles !block, can we make sure the classnames are applied in the correct order? Or we can add a 'fill' prop to the button

Reply:
This overriding turned out as not necessary anymore, because I made other solution with html inside the button. Anyway "fill" property would be needed, so I'm gonna add it

Reply:
done